openslides/users/static/templates/users/user-csv-import.html

```diff
@@ -0,0 +1,43 @@
+<h1>Import participants</h1>
+
+<div id="submenu">
+  <a ui-sref="users.user.list" class="btn btn-sm btn-default">
+    <i class="fa fa-angle-double-left fa-lg"></i>
+    <translate>Back to overview
+  </a>
+</div>
+
+<p translate>Select a CSV file to import users!</p>
+
+<p translate>Please note:</p>
+<ul>
+    <li><translate>Required comma separated values</translate>:<br>
+        <code translate>'title, first name, last name, gender, email, group id, structure level, committee, about me, comment, is active'</code>
+    <li><translate>Default groups</translate>:
+        <translate>Delegate</translate> <code>3</code>,
+        <translate>Staff</translate> <code>4</code>
+    <li translate>At least first name or last name have to be filled in. All
+      other fields are optional and may be empty.
+    <li translate>The first line (header) is ignored.
+    <li translate>Required CSV file encoding is UTF-8.
+    <li><a href="https://github.com/OpenSlides/OpenSlides/wiki/CSV-Import" translate>
+          Use the CSV example file from OpenSlides Wiki.
+        </a>
+</ul>
+
+<!-- TODO: add post url to form-->
+<form>
+  <div class="form-group">
+    <label for="InputFile" translate>CSV file:</label>
+    <input type="file" id="InputFile">
+    <p class="help-block" translate>The file has to be encoded in UTF-8.</p>
```

Comment:
The closing p-element can be removed, because:
```A p element's end tag may be omitted if the p element is immediately followed by an address, article, aside, blockquote, div, dl, fieldset, footer, form, h1, h2, h3, h4, h5, h6, header, hgroup, hr, main, nav, ol, p, pre, section, table, or ul, element, or if there is no more content in the parent element and the parent element is not an a element.```